src/irmin/conf.mli

```diff
@@ -58,11 +58,12 @@ val key :
     with a dot. [docv] is a meta-variable for representing the values of the key
     (e.g. ["BOOL"] for a boolean).
 
-    @raise Invalid_argument if the key name is not made of a sequence of ASCII
-    lowercase letter, digit, dash or underscore.
+    @raise Invalid_argument
+      if the key name is not made of a sequence of ASCII lowercase letter,
+      digit, dash or underscore.
 
-    {b Warning.} No two keys should share the same [name] as this may lead to
-    difficulties in the UI. *)
+      {b Warning.} No two keys should share the same [name] as this may lead to
+      difficulties in the UI. *)
```

Comment:
This indentation looks incorrect: I think the warning belongs to the top-level doc, not the `Invalid_argument` specifically.

![image](https://user-images.githubusercontent.com/26125912/125281508-3600c700-e30e-11eb-9f61-7dbfdb49e847.png)

Reply:
It comes from the new odoc-parser, I've opened an issue to know more about it https://github.com/ocaml-doc/odoc-parser/issues/1 but so far it looks like it's expected

Reply:
As explained in ocaml-doc/odoc-parser#1 this is expected, as this is how odoc comments are specified and parsed by odoc-parser. The only solution is to place this warning before any tag so it would not be "eaten" by any tag!

Reply:
Thanks for looking into that! I think that Odoc's HTML renderer should probably make this interpretation more obvious, but I can take that up with the Odoc maintainers directly :-)